packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -472,6 +472,11 @@ export class CreateQueryEditorStore extends QueryEditorStore {
         this.queryBuilderState.initialize(
           this.queryBuilderState.graphManagerState.graphManager.createDefaultBasicRawLambda(),
         );
+        await Promise.all([
```

Comment:
See my comment in `QueryBuilderChangeDetectionState`, we should only need to do:
`await queryBuilderState.changeDetectionState.start();` here

Reply:
Also, why do need to put this in this `else` block?